pom.xml

```diff
@@ -368,6 +368,11 @@
                 <artifactId>org.wso2.carbon.identity.handler.event.account.lock</artifactId>
                 <version>${carbon.identity.account.lock.handler.version}</version>
             </dependency>
+            <dependency>
```

Comment:
It seems the event handler has a dependency on identity governance. So this will cause cyclic dependency issues.

https://github.com/wso2-extensions/identity-event-handler-notification/blob/master/pom.xml#L190